tests/acceptance/lions/search-test.js

```diff
@@ -68,3 +69,35 @@ test('searching', function() {
     click('.lion-search');
   });
 });
+
+test('searching by age', function() {
+  expect(1);
+
+  signInAndVisit('/lions/search');
+
+  stubRequest('get', '/lions', function(request){
+    var startDate = new Date();
+    startDate.setFullYear(startDate.getFullYear() - 3);
+
+    var endDate = new Date();
+    endDate.setFullYear(endDate.getFullYear() + 3);
+
+    deepEqual(request.queryParams,
+              {dob_range_start: startDate,
+              dob_range_end: endDate},
+              'api called with correct query string for searching by gender');
+
+    return this.success({
+      _embedded: {
+        lions: []
+      }
+    });
+  });
+
+  andThen(function() {
+    var choice = dobOptions[0].label;
+    fillIn('.lg-dob-selector', choice);
+    expectElement('.lg-dob-selector:contains(' + choice + ')', 'DOB selector is filled in with 0-3 years');
```

Comment:
this must happen after another `andThen` call. `fillIn` is an [async test helper](http://emberjs.com/guides/testing/test-helpers/#toc_asynchronous-helpers), and the `expectElement` call is a sync helper.